doc/design/record_fetcher_proposal.md

```diff
@@ -0,0 +1,178 @@
+# Raw Record Representation Proposal
+
+## BLUF
+
+Upload fetched documents, having performed the minimal amount of processing. Define a manifest expressing the
+association between those document URIs and basic metadata (ID, region and record type, timestamp of collection, etc.).
+Sign that manifest and upload it. Now a logical record's components may be derived from the manifest and the now-stored
+documents the manifest describes.
+
+## Objective
+
+Define a system for tracking the associations between the various crude scrapings that describe a record. How documents
```

Comment:
doc nit: Can you split the second sentence into a new paragraph, it reads a little better